shared/common-adapters/icon.desktop.js

```diff
@@ -32,20 +33,29 @@ class Icon extends PureComponent<void, Exact<Props>, void> {
     const fontSizeHint = shared.fontSize(iconType)
 
     if (isFontIcon) {
-      const cleanStyle = {...this.props.style}
+      const cleanStyle = {
+        fontFamily: 'kb',
+        speak: 'none',
+        fontStyle: 'normal',
+        fontWeight: 'normal',
+        fontVariant: 'normal',
+        textTransform: 'none',
+        lineHeight: 1,
+        WebkitFontSmoothing: 'antialiased',
+        ...this.props.style}
       // We have to blow these styles away else FontIcon gets confused and will overwrite what it calculates
       delete cleanStyle.color
       delete cleanStyle.hoverColor
 
       return <FontIcon
         title={this.props.hint}
         style={{...globalStyles.noSelect, ...styles.icon, ...fontSizeHint, ...cleanStyle, ...(this.props.onClick ? globalStyles.clickable : {})}}
-        className={`icon-kb-${iconType}${this.props.className && ' ' + this.props.className || ''}`}
```

Comment:
instead of using the generated css file and using a classname that gives us a css :before pseudo-selector we just inject the actual charCode (which is exactly what mobile does)